frontend/src/Settings/General/GeneralSettings.js

```diff
@@ -157,12 +158,15 @@ class GeneralSettings extends Component {
                   onInputChange={onInputChange}
                 />
 
-                <UpdateSettings
-                  advancedSettings={advancedSettings}
-                  settings={settings}
-                  isMono={isMono}
-                  onInputChange={onInputChange}
-                />
+                {
+                  !isDocker &&
```

Comment:
Instead of completely not showing the section, should we instead disable everything and show a warning that updating is done via the container?

Reply:
It'll show a message in the update page if an update is available: `An update is available.  Please update your Docker image and re-create the container.`

Shall I put the same thing in here?

Reply:
I'd just show "updating is disabled on docker" or something simple on the settings page under the update header. The updates page message looks good.